packages/frontend/src/components/DashboardFilter/FilterConfiguration/index.tsx

```diff
@@ -248,12 +294,14 @@ const FilterConfiguration: FC<Props> = ({
                             internalFilterRule,
                         )
                     }
-                    onClick={() => onSave(internalFilterRule)}
+                    onClick={() => {
+                        if (!!internalFilterRule) onSave(internalFilterRule);
```

Comment:
here you can add the `setSelectedTabId(FilterTabs.SETTINGS)` 👍

Reply:
Yup, done in rebase.